src/main/java/se/sundsvall/casedata/api/model/validation/enums/Shortcode.java

```diff
@@ -0,0 +1,24 @@
+package se.sundsvall.casedata.api.model.validation.enums;
+
+import java.util.Arrays;
+
+// This should be updated to be incorporated into namespace config like in support management
+public enum Shortcode {
+
+	MEX("SBK_MEX"),
+	PRH("SBK_PARKINGPERMIT"),
+	O("UNKNOWN");
```

Comment:
What does "O" mean?

Reply:
Other. 

This class is temporary and will be removed in another 10.0 story implementing namespaceConfig (like its cousin SM) instead.